src/components/facets.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { formatLargeNumber } from '../utils';
+
+import '../styles/components/facets.scss';
+
+class Facets extends Component {
+  constructor(props) {
+    super(props);
+    this.isActive = this.isActive.bind(this);
+  }
+
+  isActive(facetName, facetValue) {
```

Comment:
could be slightly more succinct with the equivalent
```
isActive(facetName, facetValue) {
     const { selectedFacets } = this.props;
     return facetName in selectedFacets && selectedFacets[facetName].includes(facetValue);
}
```

Reply:
duh! yeah 🥇